src/cli/mod.rs

```diff
@@ -75,21 +83,33 @@ struct LoadArgs {
     /// Arithmetic field (defaults to the backend's standard field)
     #[clap(long, value_parser)]
     field: Option<String>,
+
+    /// Path for cached public parameters
+    #[clap(long, value_parser)]
+    public_params_dir: Option<Utf8PathBuf>,
+
+    /// Path for cached proofs
+    #[clap(long, value_parser)]
+    proofs_dir: Option<Utf8PathBuf>,
+
+    /// Path for cached commitments
```

Comment:
```suggestion
    /// Path to public parameters directory
    #[clap(long, value_parser)]
    public_params_dir: Option<Utf8PathBuf>,

    /// Path to proofs directory
    #[clap(long, value_parser)]
    proofs_dir: Option<Utf8PathBuf>,

    /// Path to commitments directory
```
This needs to be replicated in the duplicated code